rest/model/codegen.go

```diff
@@ -176,3 +191,89 @@ func goimports(source string) ([]byte, error) {
 		AllErrors: true, Comments: true, TabIndent: true, TabWidth: 8,
 	})
 }
+
+func CreateConversionMethods(packageName, structName string, fields ExtractedFields) ([]byte, error) {
+	pkg, err := importer.Default().Import(packageName)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to resolve package '%s'", packageName)
+	}
+	scope := pkg.Scope()
```

Comment:
Is it possible for scope to be nil? (if not you don't use this anywhere else so you can just combine it with line 201)

Reply:
added handling for nil